lib/core/src/display/scene/transform.rs

```diff
@@ -0,0 +1,84 @@
+use nalgebra::{Matrix4, Quaternion, UnitQuaternion, Vector3};
+
+// Note [Quaternion YXZ Euler angles]
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// To comply with Threejs impl, we generate a Quaternion applying rotation in
+// the order: pitch -> roll -> yaw, instead of roll -> pitch -> yaw based on
+// https://github.com/mrdoob/three.js/blob/master/src/math/Quaternion.js#L199
```

Comment:
A question - why we comply with Theejs impl? Based on this comment I understand it is not a standard way of handling it, right? Is it more performant?

Reply:
There is no performance difference. It's just a convention and we can change it whenever we want.

As I was developing this functionality based on Three.js, I used it as a visual reference and I saw something was wrong. That's when I realized the rotation order was different.